Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM registry.suse.com/bci/golang:1.21-openssl AS builder
+
+RUN set -euo pipefail; zypper -n  in --no-recommends git make ; zypper -n clean;
+
+# Create and change to the /var/lib/ directory.
+WORKDIR /var/lib
+
+# For now, we need this since we use replace directive to point to the local telemetry module in go.mod
+RUN git clone https://github.com/SUSE/telemetry
+RUN cd telemetry; make build
+
+# Copy local code to the container image.
+RUN mkdir ./telemetry-server
+COPY . ./telemetry-server
+
+RUN cd telemetry-server; make build
+
+FROM registry.suse.com/bci/bci-base:15.6
+
+RUN set -euo pipefail; zypper -n  in --no-recommends sqlite3 ; zypper -n clean;
+
+RUN useradd -r -d /var/lib/tsvc -s /sbin/nologin -c "user for telemetry-server" tsvc && \
+    mkdir -p /var/lib/tsvc && \
+    chown -R tsvc /var/lib/tsvc
+
+COPY --from=builder /var/lib/telemetry-server/server/telemetry-server/telemetry-server /usr/local/bin/telemetry-server
+COPY --from=builder /var/lib/telemetry-server/testdata/config/localServer.yaml /etc/susetelemetry/server.cfg
+
+RUN mkdir -p /tmp/telemetry/server /tmp/susetelemetry
+RUN chown -R tsvc /usr/local/bin/telemetry-server /tmp/telemetry/server /tmp/susetelemetry
+
+# Put additional files into container
+COPY entrypoint.bash .
+RUN chmod 700 ./entrypoint.bash
+
+ENTRYPOINT ["./entrypoint.bash"]
```

Comment:
See above comment about not using `.`, but rather an absolute path such as `/app/entrypoint` or similar.